File: WebServiceOfficePrivate/src/java/bonclub/office_private/web_service/IEventNotifier.java
package bonclub.office_private.web_service;


import javax.jws.WebMethod; 
import javax.jws.WebParam; 
import javax.jws.WebService; 

import bonclub.office_private.web_service.result.WsReturnValue;

@WebService
public interface IEventNotifier{
    @WebMethod
    public WsReturnValue<Boolean> notifyServerAboutSendPasswordBySms(
        @WebParam(name="phoneNumber") String phoneNumber);
    @WebMethod
    public WsReturnValue<Boolean> sendLoginAndPasswordByMail(
        @WebParam(name="boncardNumber") String boncardNumber);
    @WebMethod
    public WsReturnValue<Boolean> sendLoginAndPasswordByMailByEmail(
        @WebParam(name="email") String email);

}

File: WebServiceOfficePrivate/src/php/IEventNotifier.php
<?php

class ieventnotifier{
    private $proxy;

    public function __construct($pUrl){
        $this->proxy = new SoapClient($pUrl);
    }

    public function notifyServerAboutSendPasswordBySms($phoneNumber){
        $userData->phoneNumber=new SoapVar(urlencode($phoneNumber), XSD_STRING);
        return $this->proxy->__soapCall("notifyServerAboutSendPasswordBySms", array($userData) );
    }
    public function sendLoginAndPasswordByMail($boncardNumber){
        $userData->boncardNumber=new SoapVar(urlencode($boncardNumber), XSD_STRING);
        return $this->proxy->__soapCall("sendLoginAndPasswordByMail", array($userData) );
    }
    public function sendLoginAndPasswordByMailByEmail($email){
        $userData->email=new SoapVar(urlencode($email), XSD_STRING);
        return $this->proxy->__soapCall("sendLoginAndPasswordByMailByEmail", array($userData) );
    }
}

?>